purchase_allowed_product/models/purchse_order.py

```diff
@@ -0,0 +1,9 @@
+# © 2017 Today Mourad EL HADJ MIMOUNE @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
```

Comment:
can we take the opportunity of the migration to fix this file's name (don't forget to change it in __init__.py)